Tools/cases_generator/optimizer_generator.py

```diff
@@ -27,7 +27,7 @@
 from stack import Stack, SizeMismatch, UNUSED
 
 DEFAULT_OUTPUT = ROOT / "Python/optimizer_cases.c.h"
-DEFAULT_ABSTRACT_INPUT = ROOT / "Python/optimizer_bytecodes.c"
+DEFAULT_ABSTRACT_INPUT = (ROOT / "Python/optimizer_bytecodes.c").absolute().as_posix()
```

Comment:
This change is needed because of `parser/prettify_filename` which works only with an `str` type.